pkg/router/template/configmanager/haproxy/testing/haproxy.go

```
@@ -107,10 +107,10 @@ func (p *fakeHAProxy) Commands() []string {
 
 func (p *fakeHAProxy) Start() {
 	started := make(chan bool)
-	go func() {
+	go func() error {
 		listener, err := net.Listen("unix", p.socketFile)
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)